ods_ci/tasks/Resources/RHODS_OLM/pre-tasks/oc_is_operator_installed.robot

```diff
@@ -71,7 +73,7 @@ Check A RHODS Family Operator Is Installed
       ${result} =  Run Keyword And Return Status
       ...  Oc Get  kind=ClusterServiceVersion  namespace=${namespace}  name=${current_csv_name}
   ELSE
-      ${result} =  Set Variable    False
+      ${result} = Set Variable    False
```

Comment:
@tarukumar - is this changed on purpose? Similarly as the other whitespace changes. At least this one is causing dry-run to fail again. My understanding is this change is unwanted but I may be missing something.